docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -364,29 +364,29 @@ use the following rules:
 Impersonation rules
```

Comment:
Let's remove `doc: ` commit message prefix. We don't have such convention.

Reply:
Thanks, fixed